### ISSUE TITLE
Docker with Apache Ignite crash 1.8.0_332-b09

### ISSUE THREAD

**Issue by malenkymuk** [adoptium/containers#294]
Hello, i've got this calling usual java lambda expression in my Apache Ignite docker container.

# A fatal error has been detected by the Java Runtime Environment:
#
#  SIGSEGV (0xb) at pc=0x00007fcc94197f24, pid=7, tid=0x00007fcb8eab1700
#
# JRE version: OpenJDK Runtime Environment (8.0_332-b09) (build 1.8.0_332-b09)
# Java VM: OpenJDK 64-Bit Server VM (25.332-b09 mixed mode linux-amd64 compressed oops)
# Problematic frame:
# V  [libjvm.so+0x470f24]  CompilationPolicy::can_be_compiled(methodHandle, int)+0x34
#
# Core dump written. Default location: /opt/ignite/core or core.7
#
# An error report file with more information is saved as:
# /tmp/hs_err_pid7.log
#
# If you would like to submit a bug report, please visit:
#   https://github.com/adoptium/adoptium-support/issues

**Comment by karianna** [comment]
Please try with the latest 8u352 when that is released this week.

**Comment by malenkymuk** [comment]
Yes, i've same crash on 1.8.0_352-b08 immidietly, also we've got this on Amazon jdk. So, i little confused.

**Comment by karianna** [comment]
@malenkymuk Interesting - are you able to upload the files:

Core Default location: /opt/ignite/core or core.7
An error report file with more information is saved as: /tmp/hs_err_pid7.log

**Comment by karianna** [comment]
Also does this occur outside of a container (just on a regular VM or bare metal host).

Ohhhh that reminds me, what version of Docker are you running?

**Comment by malenkymuk** [comment]
Its hard to reporoduce it ouside of container. It seems, that was provoked by Apache Ignite state due to serializing/unserializing proccess. After code changing its disappeared.
Docker Engine:
  Version:          20.10.17
  API version:      1.41 (minimum version 1.12)
  Go version:       go1.17.11
  Git commit:       a89b842
  Built:            Mon Jun  6 23:03:33 2022
  OS/Arch:          linux/amd64
  Experimental:     false

**Comment by karianna** [comment]
Hmm, we probably need those files then:

Core Default location: /opt/ignite/core or core.7
An error report file with more information is saved as: /tmp/hs_err_pid7.log
